plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java

```diff
@@ -85,6 +87,13 @@ private boolean validateLocalFields() {
       return false;
     }
     
+    File parent = getLocationPath().toFile();
+    File projectDirectory = new File(parent, getProjectName());
```

Comment:
Agreed there's a bug, but if there's a project name shouldn't we use that in determin where to put the project? I.e. it sounds to me like if the user selects

/usr/local/tmp with a project name of sample, then we should create the project in 

/usr/local/tmp/sample